openvdb_houdini/houdini/ParmFactory.h

```diff
@@ -532,6 +532,11 @@ class OPENVDB_HOUDINI_API OpPolicy
     /// factory.@link OpFactory::english() english()@endlink.
     virtual std::string getName(const OpFactory& factory, const std::string& english);
 
+    /// @brief Return a label name for the operator defined by the given factory.
+    /// @details In this base class implementation, this method simply returns
+    /// factory.@link OpFactory::english() english()@endlink.
+    virtual std::string getLabelName(const OpFactory&);
```

Comment:
I don't anticipate any issues with this, but it's safest to add new virtual functions to the end of the class.

Reply:
Fixed.